takealot.user.js

```diff
@@ -0,0 +1,520 @@
+// @require        https://raw.githubusercontent.com/murdos/musicbrainz-userscripts/master/lib/mbimport.js
+// @require        https://raw.githubusercontent.com/murdos/musicbrainz-userscripts/master/lib/logger.js
+// @require        https://raw.githubusercontent.com/murdos/musicbrainz-userscripts/master/lib/mblinks.js
+// @require        https://raw.githubusercontent.com/murdos/musicbrainz-userscripts/master/lib/mbimportstyle.js
+// @icon           https://raw.githubusercontent.com/murdos/musicbrainz-userscripts/master/assets/images/Musicbrainz_import_logo.png
+// @grant          none
+// ==/UserScript==
+
+// need to do button and paste next
+// needto look at artist it picked up track1 as the artist name.
+
+
+// prevent JQuery conflicts, see http://wiki.greasespot.net/@grant
+this.$ = this.jQuery = jQuery.noConflict(true);
+
+if (!unsafeWindow) unsafeWindow = window;
+
+var DEBUG = true;
+//DEBUG = true;
+if (DEBUG) {
+  LOGGER.setLevel('debug');
+}
+
+////////////////////////////////////////////////////////////////////////////////////////////////////////
+
+/*
+ * Test cases:
+ * - http://www.takealot.com/theuns-jordaan-roeper-cd/PLID17284867
+ */
+
+
+
+$(document).ready(function() {
+	LOGGER.info("Document Ready & Takealot Userscript executing");
+	var TakealotRelease = ParseTakealotPage();	
+	insertMBSection(TakealotRelease);
+});
+
+////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////
+//                             Insert MusicBrainz section into Discogs page                                           //
+////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////
+
+function insertMbUI(mbUI) {
+  var e;
+  if ((e = $("div.section.box-summary")) && e.length) {
+    e.after(mbUI);
+  } else if ((e = $('#buybox')) && e.length) {
+    e.before(mbUI);
+  } else if ((e = $("div.section.more-choices")) && e.length) {
+    e.before(mbUI);
+  }
+}
+
+
+
+
+// Insert links in Takealot page
+function insertMBSection(release) {
+   LOGGER.debug("insertMBsection Firing");
+
+    var mbUI = $('<div class="section musicbrainz"><h3>MusicBrainz</h3></div>').hide();
+
+    if (DEBUG) mbUI.css({'border': '1px dotted red'});
+
+    var mbContentBlock = $('<div class="section_content"></div>');
+    mbUI.append(mbContentBlock);
+
+    if (release.maybe_buggy) {
+      var warning_buggy = $('<p><small><b>Warning</b>: this release has perhaps a buggy tracklist, please check twice the data you import.</small><p').css({'color': 'red', 'margin-top': '4px', 'margin-bottom': '4px'});
+      mbContentBlock.prepend(warning_buggy);
+    }
+
+    // Form parameters
+    var edit_note = MBImport.makeEditNote(window.location.href, 'Takealot');
+	  LOGGER.debug("*** Edit Note: ",edit_note);
+    var parameters = MBImport.buildFormParameters(release, edit_note);
+   LOGGER.debug("***Form parameters");
```

Comment:
Please take care of indentation, you may use an online indenter to get a clean js file.